packages/flutter_tools/test/host_cross_arch.shard/macos_content_validation_test.dart

```diff
@@ -164,6 +164,20 @@ void main() {
         ),
       );
 
+      final File flutterMacOSFramework = fileSystem.file(
+        fileSystem.path.join(
+          outputApp.path,
+          'Contents',
+          'Frameworks',
+          'FlutterMacOS.framework',
+        ),
+      );
```

Comment:
`outputFlutterFramework` is the same thing immediately above, so you should be able to use that.  And it's a directory, not a file.